arangod/GeneralServer/H2CommTask.cpp

```diff
@@ -154,7 +161,10 @@ template <SocketType T>
   H2CommTask<T>* me = static_cast<H2CommTask<T>*>(user_data);
   auto it = me->_streams.find(stream_id);
   if (it != me->_streams.end()) {
-    it->second->statistics.SET_WRITE_END();
+    Stream& strm = *it->second;
+    if (strm.response && strm.response->statistics) {
```

Comment:
```suggestion
    if (strm.response) {
```
No need to check `statistics`, this is implicitly done in `SET_WRITE_END()` & co.